tests/mcaptcha/default.nix

```diff
@@ -0,0 +1,103 @@
+{modules, ...}: let
+  port = 7000;
+  urlRoot = "http://localhost:${builtins.toString port}";
+in {
+  name = "mcaptcha tests";
+
+  nodes.create_locally = {pkgs, ...}: {
+    imports = [
+      modules.default
+      modules.mcaptcha
+    ];
+
+    services.mcaptcha.enable = true;
+    services.mcaptcha.settings.server.port = port;
+    services.mcaptcha.server.cookieSecretFile = pkgs.writeText "cookie-secret" "mcaptcha-cookie-secret-dm0tdGVzdC1ydW4tbWNhcHRjaGEtdGVzdHM";
+    services.mcaptcha.captcha.saltFile = pkgs.writeText "salt" "asdl;kjfhjawehfpa;osdkjasdvjaksndfpoanjdfainsdfaijdsfajlkjdsaf;ajsdfweroire";
+    services.mcaptcha.database.createLocally = true;
+    services.mcaptcha.redis.createLocally = true;
+
+    # the following two lines are for debugging
+    networking.firewall.enable = false;
+    services.mcaptcha.settings.server.ip = "0.0.0.0";
```

Comment:
In which sense do you mean "for debugging"? If you mean only for interactive builds of the test, then please put these configuration in [`interactive.nodes.<name>`](https://nixos.org/manual/nixos/stable/#test-opt-interactive).